From e3b645467ad19fe0ed381c9cb5b14c0345bb598f Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Fri, 10 Nov 2017 16:19:58 -0500
Subject: [PATCH] NET-381 use triton-origin for booter/dhcpd TRITON-13 service
 configure.sh scripts are of inconsistent strictness NET-380 dhcpd service is
 only accidenlty enabled

---
 boot/configure.sh | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/boot/configure.sh b/boot/configure.sh
index bfae9c7..67fa751 100755
--- a/boot/configure.sh
+++ b/boot/configure.sh
@@ -7,17 +7,19 @@
 #
 
 #
-# Copyright (c) 2015, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+set -o errexit
+set -o pipefail
 set -o xtrace
 
 echo "Importing dhcpd manifest"
 /usr/sbin/svccfg import /opt/smartdc/booter/smf/manifests/dhcpd.xml
 
 echo "Enabling dhcpd service"
-/usr/sbin/svcadm enable smartdc/site/dhcpd
+/usr/sbin/svcadm enable smartdc/application/dhcpd
 
 echo "Importing tftpd manifest"
 /usr/sbin/svccfg import /opt/smartdc/booter/smf/manifests/tftpd.xml
@@ -29,9 +31,9 @@ echo "Configuring nginx"
 cp /opt/smartdc/booter/etc/nginx.conf /opt/local/etc/nginx/nginx.conf
 
 echo "Importing nginx manifest"
-/usr/sbin/svccfg import /opt/local/share/smf/nginx/manifest.xml
+/usr/sbin/svccfg import /opt/local/lib/svc/manifest/nginx.xml
 
 echo "Enabling nginx service"
-/usr/sbin/svcadm enable network/nginx
+/usr/sbin/svcadm enable pkgsrc/nginx
 
 exit 0
-- 
2.21.0

